tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/core/planner.h

```diff
@@ -40,13 +40,15 @@ namespace tesseract_planning
 class MotionPlanner
 {
 public:
+  using Ptr = std::shared_ptr<MotionPlanner>;
+  using ConstPtr = std::shared_ptr<const MotionPlanner>;
   /** @brief Construct a basic planner */
   MotionPlanner(std::string name) : name_(std::move(name)) {}
   virtual ~MotionPlanner() = default;
-  MotionPlanner(const MotionPlanner&) = default;
-  MotionPlanner& operator=(const MotionPlanner&) = default;
-  MotionPlanner(MotionPlanner&&) = default;
-  MotionPlanner& operator=(MotionPlanner&&) = default;
+  MotionPlanner(const MotionPlanner&) = delete;
+  MotionPlanner& operator=(const MotionPlanner&) = delete;
+  MotionPlanner(MotionPlanner&&) = delete;
+  MotionPlanner& operator=(MotionPlanner&&) = delete;
```

Comment:
Do we need a clone now? Are we using this anywhere?

Reply:
Probably should add a clone, but I think right now we are creating shared and unique pointers which get passed around. It would be good to add a clone but nothing is currently using this library in that way.

Reply:
I am adding the clone method now.

Reply:
@mpowelson I added the clone method.